sycl/source/detail/graph_impl.hpp

```diff
@@ -313,6 +340,16 @@ class graph_impl {
     MInorderQueueMap[QueueWeakPtr] = Node;
   }
 
+  /// Prints contents of graph to text file in DOT format
+  /// \param GraphName is a string appended to the output file name
+  void printGraphAsDot(const char *GraphName);
+
+  /// Checks is the graph_impl of Graph has a similar structure to
```

Comment:
We're adding new symbols here so the ABI symbols test needs updated. From ` abi/sycl_symbols_linux.dump`
```
The following symbols are new to the object file:

_ZN4sycl3_V13ext6oneapi12experimental6detail24modifiable_command_graph19hasSimilarStructureERKNS3_13command_graphILNS3_11graph_stateE0EEEi
_ZN4sycl3_V13ext6oneapi12experimental6detail24modifiable_command_graph15printGraphAsDotEPKc

```